src/pages/partner/index.md

```diff
@@ -41,173 +44,149 @@ The following workflow diagram describes the steps in the partner registration p
 
 ![Partner Integration Workflow](resources/Partner_Integration_Workflow.png)
 
-1. **Offline approval**: You can reach out to your contact in the Adobe’s globalization team. 
-
-
-    Your contact point will work with you to get your firm’s details registered in the localization partners team. After necessary approvals, a contract would be signed. Based on the data you provide, such as Org Id, the name of organization, and services you will provide, an org would be created for you. This org will have predefined entitlements to the globalization content service. 
-2. **Get an IMS token:** The Globalization Content Service team procures an IMS bearer token for your firm. Use this token to get authenticated through the Adobe I/O console. 
-3. **Log on the Adobe I/O Console**: Use the IMS token to log in Adobe I/O console. Once verified, you can access the products and services based on the permissions granted to you. 
-4. **Subscribe to APIs**: You can subscribe to the Adobe I/O Globalization Content Service APIs to manage projects, tasks, and assets. 
-5. **Subscribe to events**: Using Adobe I/O journaling APIs, subscribe to events and get near real-time communication. Even if you have thousands of tasks needing localization every week, the event journal will help you get info as soon as tasks are being triggered at the product end. Based on task information received in events, retrieve assets, localize them, preview quality, and send final translated assets back to the product through the Adobe I/O Globalization Content Service APIs. 
+1. **Get an IMS token:** The Globalization Content Service team procures an IMS bearer token for your firm. Use this token to get authenticated through the Adobe I/O console.
```

Comment:
@varunutreja Is this not self serve. Anyone can come and create an account to use with the dev console. Do we have to procure the token?

Reply:
No we don't need token. Removed this line.